src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -110,16 +113,17 @@ public function initialize(array $config)
      * @param string $model the model field value
      * @return void
      */
-    public function setupFieldAssociations($fields, $table, $model)
+    public function setupFieldAssociations($fields, $table, $model, $strategy)
     {
+        $targetAlias = Inflector::slug($table);
```

Comment:
Actually yes I'm expecting that, since it should respect any possible Table value, e.g. `'translationTable' => 'Company/Plugin.I18n'`